frontend/src/pages/CreateProjectPage.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Trans, useTranslation } from 'react-i18next';
+import { shallowEqual, useSelector } from 'react-redux';
+import { Redirect } from 'react-router-dom';
+import { paths } from '../routers/paths';
+import { ModalContent } from '../components/modals/modalparts/ModalContent';
+import { ModalHeader } from '../components/modals/modalparts/ModalHeader';
+import { Footer } from '../components/Footer';
+import { RootState } from '../redux/types';
+import { userInfoSelector } from '../redux/user/selectors';
+import { UserInfo } from '../redux/user/types';
+import css from './CreateProjectPage.module.scss';
+
+const classes = classNames.bind(css);
+
+export const CreateProjectPage = () => {
+  const { t } = useTranslation();
+  const userInfo = useSelector<RootState, UserInfo | undefined>(
+    userInfoSelector,
+    shallowEqual,
+  );
+
+  if (!userInfo) return <Redirect to={paths.loginPage} />;
+
+  return (
+    <>
+      <div className={classes(css.container)}>
+        <ModalHeader>{t('Greet user', { userInfo })}</ModalHeader>
+        <ModalContent>
+          <div className={classes(css.about)}>
+            <Trans i18nKey="No projects yet" />
+          </div>
+          <div className={classes(css.body)}>
+            <Trans i18nKey="First project info 1/2" />
+          </div>
+          <div className={classes(css.body)}>
+            <Trans i18nKey="First project info 1/2" />
```

Comment:
Looks like a typo: the first info is repeated twice
```suggestion
            <Trans i18nKey="First project info 2/2" />
```